process/dd-review-template.md

```diff
@@ -24,19 +24,19 @@ As a secondary goal, it is in the interests of the broader CNCF ecosystem that t
 /* Use appropriate Section */
 
 ### Sandbox Graduation (Exit Requirements)
-1. Document that it is being used successfully in production by at least three independent end users which with focus on adequate quality and scope defined.
+1. Document that it is being used successfully in production by at least three independent adopters which with focus on adequate quality and scope defined.
 2. Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
 3. Demonstrate a substantial ongoing flow of commits and merged contributions.
 
 ### Incubating Stage Graduation (Exit Requirements)
-1. Document that it is being used successfully in production by at least three independent end users which with focus on adequate quality and scope defined.
+1. Document that it is being used successfully in production by at least three independent adopters which with focus on adequate quality and scope defined.
```

Comment:
For graduation criteria, I'd suggesting we clearly claim "direct adopters" instead of "adopters". For example, as a cloud vendor which sell a certain project, it only counts 1 adopter for that project, instead of N adopters (assume there are N customers of that product). This applies to consultant companies as well.

Reply:
👍🏻